neuralmagic/benchmarks/configs/minimal_test.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
This is useful to quickly test the benchmarking workflow by triggering manual jobs